content/post/cheers-to-small-beginnings.md

```diff
@@ -0,0 +1,55 @@
+
+### All aboard
+
+We conducted some research to reach base one. It was stunning to see everybody excited speaking out loud about their beliefs and the passion that drives them. “Honest work for fun on funny things” it’s a glimpse on how we sparkled team’s spirit all over this journey. We knew from the start that albeit design’s written word was still immature, it shouted strong diverse aesthetic: “I don't think a nile crocodile is serious. It's always smiling :)” and so the journey from individual to unified vision began.
+
+We believe that design is a team sport, and it happens when there is such knowledge management, its transfer and openness within a team that it raises the standards of the critical environment. Design must capacitate everyone involved with the power to surface the value of collective truth, driving the momentum far and near the birthing of design’s written word towards its visual translation.
+
+### Cookin’ on three burners
+
+Genuine design happens when we acknowledge that design without rationale is a mere aesthetic exercise. Moreover, the urge for better experiences is hinged on better people not on better design. Bringing it all together, we genuinely praise people above all.
+
+The visual translation started fuelled with qualitative data, quantitative data and our company culture. We first started looking for some typography options, and in the process, we stumble upon [Free Font Ballon](https://www.behance.net/gallery/55482949/Free-Font_Balloon) by the incredibly talented [Aga Magdziak](https://www.behance.net/aga735d). After a couple of minutes we found out that there was more to Aga than meets the eye. Along with the typeface, Aga created a huge set of visual artifacts that caught our attention. We like to think of what we discovered as open source design, and we deeply believe in it. In the end we came up with a design lexicon - a branch of knowledge - that empowered us create the Brand Persona, and it was data-oriented all the way.
+
+![{d}ali Lexicon](/post/cheers-to-small-beginnings/{d}ali-lexicon.png)
+
+### Claws that can grab a dinosaur
+
+With a lexicon, we had all we needed to create the Brand Persona’s visual translation. After vivid care on content analysis we chose to give birth to {d}ali, a quite particular parrot. A parrot has the distinct characteristic of easily learn languages, and that pretty much stroked our agenda. Then, we had to make sure that our Brand Persona personality and traits would reflect on {d}ali aesthetics. Its claws translate our brand’s trustworthiness, its tail translate our brand’s enthusiasm and {d}ali’s head is all about being genuine, mainly because it gathers characteristics from several parrots, each one reinforcing our rational. And this, is how {d}ali was born.
```

Comment:
I feel that "with a lexicon" is counter-English but cannot suggest a better alternative.

Reply:
a quite particular parrot -> a quite special parrot

Reply:
of easily learning languages

Reply:
Its claws -> It's claws

Reply:
its tail -> it's tail

Reply:
our rational -> our rationale ?

Reply:
And this, is how {d}ali was born.

This is how {d}ali was born.

Reply:
Thanks!

All changed, except:

> Its claws -> It's claws

its claws = the parrots claws
it's claws = it is parrot claws